db/data/regions/_default/fi.yml

```diff
@@ -94,7 +95,8 @@ fi:
             too_long: Kaupungin nimi on liian pitkä (enintään 255 merkkiä)
             contains_html_tags: Paikkakunta ei voi sisältää HTML-tunnisteita.
             contains_emojis: Kaupunki ei voi sisältää emojeita
-            contains_mathematical_symbols: Kaupunki ei voi sisältää matemaattisia symboleja
+            contains_mathematical_symbols: Postitoimipaikka ei voi sisältää matemaattisia
+              symboleja
```

Comment:
🤔 I think this should use `Kaupunki` as the other messages related to this field do, not `Postitoimipaikka`?